### PR TITLE
fix press and hold and maximized state of remote views on windows

### PR DESCRIPTION
**Pull request by SlrG**
This PR should fix the issue #775 where the press and hold feature is closing its remote view window automatically on windows systems. Also it fixes the maximized state of the remote view windows, which was not working correctly on windows systems, too.

### CODE REVIEW

File: master/src/ComputerMonitoringWidget.h [hunk 1]
```diff
@@ -82,6 +83,7 @@ class ComputerMonitoringWidget : public FlexibleListView, public ComputerMonitor
 	bool m_ignoreMousePressAndHoldEvent{false};
 	bool m_ignoreWheelEvent{false};
 	bool m_ignoreResizeEvent{false};
+	int m_ignoreNumberOfMouseEvents;
```
Comment (tobydox) [inline master/src/ComputerMonitoringWidget.h:86]: Please initialize the member with 0.

File: plugins/remoteaccess/RemoteAccessWidget.cpp [hunk 2]
```diff
@@ -321,10 +321,10 @@ RemoteAccessWidget::RemoteAccessWidget( const ComputerControlInterface::Pointer&
 	connect( m_vncView, &VncViewWidget::mouseAtBorder, m_toolBar, &RemoteAccessWidgetToolBar::appear );
 	connect( m_vncView, &VncViewWidget::sizeHintChanged, this, &RemoteAccessWidget::updateSize );
 
-	showMaximized();
+	setWindowState(Qt::WindowMaximized);
 	VeyonCore::platform().coreFunctions().raiseWindow( this, false );
 
-	showNormal();
+	show();
```
Comment (tobydox) [inline plugins/remoteaccess/RemoteAccessWidget.cpp:328]: Can you please separate the changes to RemoteAccessWidget into a separate commit?

File: master/src/ComputerMonitoringWidget.cpp [hunk 3]
```diff
@@ -293,19 +294,21 @@ void ComputerMonitoringWidget::runMousePressAndHoldFeature( )
 		selectedInterfaces.first()->hasValidFramebuffer() )
    {
 		m_ignoreMousePressAndHoldEvent = true;
+		m_ignoreNumberOfMouseEvents = 3;
```
Comment (tobydox) [inline master/src/ComputerMonitoringWidget.cpp:297]: Please define a `static constexpr auto IgnoredNumberOfMouseEventsWhileHold = 3` in ComputerMonitoringWidget.h to eliminate this anonymous value in the code.

**Comment by tobydox** [comment]
Except for the commented minor issue, everything looks good, thank you very much! Maybe a short explanation on why exactly 3 mouse events have to be ignored would be nice in the commit description.

**Comment by SlrG** [comment]
I'll try to implement the requested changes. 

As to why three events need to be ignored, I'm not perfectly sure myself. I started with one event, which worked fine on my newer development system. But I was surprised, it still failed on my older testing system. Increasing to two worked often, but only going to three did solve the problem, there too. Could it be the performance of the systems? I don't know.

**Comment by SlrG** [comment]
What would you do to show resize the ComputerZoomWidget window to be exactly the same size as the master and on the same position instead of moving to the masters x,y position and maximizing? Using 
`setGeometry( masterWindow->x(), masterWindow->x(), masterWindow->width(), masterWindow->height() );`
does not work correctly on windows while on linux it does. The resulting position on windows is off to the top, left, I believe.
